packages/mocks/tsconfig.json

```diff
@@ -7,4 +7,5 @@
     "outDir": "./dist",
     "rootDir": "./src"
   },
+  "references": [{ "path": "../types" }]
```

Comment:
what does that do ?
mocks is a library we use from the plugin, we dont use it directly, AFAIK this will not do anything here ?

Reply:
@sjd78 ^^ ?

Reply:
please remove the unneeded line, because it's "dead" code.